client/packages/inventory/src/Stocktake/DetailView/Toolbar.tsx

```diff
@@ -47,6 +55,41 @@ export const Toolbar = () => {
   const setItemFilter = (itemFilter: string) =>
     updateQuery({ itemCodeOrName: itemFilter });
 
+  const {
+    isOpen: reduceIsOpen,
+    onClose: onCloseReduce,
+    onOpen: onOpenReduce,
+  } = useEditModal();
+  const {
```

Comment:
Hmm i wonder if we need to destructure these ones ?

const reduce = useEditModal();
const changeLocation = useEditModal();

// ..

reduce.isOpen // etc